### PR TITLE
cn update

### PR DESCRIPTION
**Pull request by wangzhao0217**
- **update color band for CN**
- **add funs and update web**
- **update notes**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Cool, good to see! Will test.

File: index.html [hunk 1]
```diff
@@ -236,11 +236,23 @@ <h2>Layer controls</h2>
...
 			<button class="accordion">Coherent network</button>
 			<div class="panel">
-				<p>Indicates a suggested order in which the network should grow. Group 1 is highest priority.</p>
-				<p><label><input type="checkbox" class="showlayer" data-layer="coherentnetwork" value="false"> Show layer</label></p>
+				<p>Indicates a suggested order in which the network should grow. Group 1 represents the smallest network, and Group 4 represents the largest, with networks growing in size from 1 to 4.</p>
+				
+				<!-- Add dataset selector dropdown -->
+				<div>
+					<label for="networkSelector">Select Network Dataset:</label>
+					<select id="networkSelector" onchange="updateNetworkSource()">
```
Comment (Robinlovelace) [inline index.html:244]: OK I see the logic here.

Comment (Robinlovelace) [reply index.html:244]: I still think we should only have 1 .pmtiles object.

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
See comments above. Let's wait for @mvl22 to comment when back. In the meantime I'll try to update the datasets in a separate PR.

**Comment by mvl22** [comment]
I'll review this shortly. It's not really the best way to do this, but I can hopefully make the documentation clearer for this scenario anyway.

PS The layer management stuff is in a bit of a state of flux at present, with the legend stuff not always fully auto-generated from a definition for instance.

> I still think we should only have 1 .pmtiles object.

Yes, there should be a single dataset with different values in it. Is that on its way?

**Comment by Robinlovelace** [comment]
Let's revive this conversation in after some changes on the backend side. Great job @wangzhao0217 getting your head round the .js code, this PR works, but I think we need to think over the cn implementation, maybe in a call with @mvl22, before updating it.

**Pull request closed without merging**